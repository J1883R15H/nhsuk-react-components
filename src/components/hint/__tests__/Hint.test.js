import React from 'react';
import { shallow } from 'enzyme';
import Hint from '../hint';

describe('Hint', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Hint>Hint</Hint>);
    expect(wrapper).toMatchSnapshot();
    wrapper.unmount();
  });
});
