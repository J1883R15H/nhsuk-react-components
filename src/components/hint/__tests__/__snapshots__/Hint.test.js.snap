// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hint matches snapshot 1`] = `
<span
  className="nhsuk-hint"
>
  Hint
</span>
`;
